crates/wash-cli/src/dev.rs

```diff
@@ -152,25 +319,482 @@ impl KnownDep {
+                        ("wasi", "http", "incoming-handler", _) => {
+                            link_property
+                                .source
+                                .get_or_insert(Default::default())
+                                .config
+                                .push(ConfigProperty {
+                                    name: "default".into(),
+                                    properties: Some(HashMap::from([(
+                                        "address".into(),
+                                        DEFAULT_INCOMING_HANDLER_ADDRESS.into(),
+                                    )])),
+                                });
+                        }
+                        ("wasmcloud", "messaging", "handler", _) => {
+                            link_property
+                                .source
+                                .get_or_insert(Default::default())
+                                .config
+                                .push(ConfigProperty {
+                                    name: "default".into(),
+                                    properties: Some(HashMap::from([(
+                                        "subscriptions".into(),
+                                        DEFAULT_MESSAGING_HANDLER_SUBSCRIPTION.into(),
+                                    )])),
+                                });
+                        }
+                        _ => {}
+                    }
+
+                    // TODO: Check for on-dep overrides/additional configuration to add to the link
+
+                    let link_trait = wadm_types::Trait {
+                        trait_type: "link".into(),
+                        properties: TraitProperty::Link(link_property),
+                    };
+
+                    // Add the trait
+                    dep_component
+                        .traits
+                        .get_or_insert(Vec::new())
+                        .push(link_trait);
+                }
+            }
+
+            // Add the dependency component after we've made necessary links
+            components.push(dep_component);
+        }
+
+        // Add the application component after we've made necessary links
+        components.push(app_component);
+
+        manifests.push(Manifest {
+            api_version: "0.0.0".into(),
+            kind: "Application".into(),
+            metadata: Metadata {
+                name: app_name,
+                annotations: BTreeMap::from([("version".into(), "v0.0.0".into())]),
+                labels: BTreeMap::new(),
+            },
+            spec: Specification {
+                components,
+                policies: Vec::from([Policy {
+                    name: "nats-kv".into(),
+                    policy_type: "policy.secret.wasmcloud.dev/v1alpha1".into(),
+                    properties: BTreeMap::from([("backend".into(), "nats-kv".into())]),
+                }]),
```

Comment:
I noticed this in my generated manifest and was a little confused 🤔 I assume this is a placeholder for future secrets integration but if not I think we can remove